### DISCUSSION TITLE
Students Id has 2 Sides

### DISCUSSION THREAD

**Discussion by LeonBoettcher** [community/community#64965] [GitHub Education]
### Hi everyone,

My Students ID has 2 side, one with all informations about me and my school and the back has the seal of validity on it. I dont have a school e-mail so i can only use my cam, which can only make one photo. t says either it cant see the school name on it or it not seems to be valid.

**Comment by arafajs** [comment]
@LeonMonkeygamer 
Take a picture of the 2 Sides of ID and put them together in one image on the phone. and when the verifying by the camera. scan your phone screen instead of your ID.
